io.github.justinrdonnelly.bouncer.json

```diff
@@ -0,0 +1,69 @@
+{
+    "id": "io.github.justinrdonnelly.bouncer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+    ],
+    "command": "io.github.justinrdonnelly.bouncer",
+    "finish-args": [
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--system-talk-name=org.freedesktop.NetworkManager",
+        "--system-talk-name=org.fedoraproject.FirewallD1"
+    ],
+    "build-options": {
+        "env": {}
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "libportal",
+            "buildsystem": "meson",
+            "config-opts": [
+                "--libdir=/app/lib",
```

Comment:
```suggestion
```

Reply:
no you don't. Use the flatpak-builder provided as a flatpak, since Ubuntu can't be bothered.